cmd/api/v1/audit.go

```diff
@@ -0,0 +1,180 @@
+package v1
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+	"time"
+
+	"connectrpc.com/connect"
+	apiv1 "github.com/metal-stack-cloud/api/go/api/v1"
+	"github.com/metal-stack-cloud/cli/cmd/config"
+	"github.com/metal-stack-cloud/cli/cmd/sorters"
+	"github.com/metal-stack/metal-lib/pkg/genericcli"
+	"github.com/metal-stack/metal-lib/pkg/genericcli/printers"
+	"github.com/metal-stack/metal-lib/pkg/pointer"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"google.golang.org/protobuf/types/known/timestamppb"
+)
+
+type audit struct {
+	c *config.Config
+}
+
+func newAuditCmd(c *config.Config) *cobra.Command {
+	a := &audit{
+		c: c,
+	}
+
+	cmdsConfig := &genericcli.CmdsConfig[*apiv1.AuditServiceGetRequest, *apiv1.AuditServiceGetRequest, *apiv1.AuditTrace]{
+		BinaryName:      config.BinaryName,
+		GenericCLI:      genericcli.NewGenericCLI[*apiv1.AuditServiceGetRequest, *apiv1.AuditServiceGetRequest, *apiv1.AuditTrace](a).WithFS(c.Fs),
```

Comment:
Use `any` in case there is  no such request type available.

```suggestion
	cmdsConfig := &genericcli.CmdsConfig[any, any, *apiv1.AuditTrace]{
		BinaryName:      config.BinaryName,
		GenericCLI:      genericcli.NewGenericCLI(a).WithFS(c.Fs),
```